src/lib/spinel/spinel_driver.hpp

```diff
@@ -180,6 +180,19 @@ class SpinelDriver : public otSpinelDriver, public Logger
                         const char       *aFormat,
                         va_list           aArgs);
 
+    /*
+     * Sends a spinel command without arguments to the co-processor.
+     *
+     * @param[in] aCommand    The spinel command.
+     * @param[in] aKey        The spinel property key.
+     * @param[in] aTid        The spinel transaction id.
+     *
+     * @retval  OT_ERROR_NONE           Successfully sent the command through spinel interface.
+     * @retval  OT_ERROR_INVALID_STATE  The spinel interface is in an invalid state.
+     * @retval  OT_ERROR_NO_BUFS        The spinel interface doesn't have enough buffer.
+     *
+     */
+    otError SendCommand(uint32_t aCommand, spinel_prop_key_t aKey, spinel_tid_t aTid);
```

Comment:
nit: add empty line after
```suggestion
     */
    otError SendCommand(uint32_t aCommand, spinel_prop_key_t aKey, spinel_tid_t aTid);

```